components/brave_wallet/browser/solana_message_unittest.cc

```diff
@@ -77,6 +77,19 @@ TEST(SolanaMessageUnitTest, Serialize) {
   };
   EXPECT_EQ(message_bytes.value(), expected_bytes);
 
+  // Deserialize and serialize message again should have the same byte array.
+  auto deserialized_message = SolanaMessage::Deserialize(*message_bytes);
+  ASSERT_TRUE(deserialized_message);
+  signers.clear();
+  auto serialized_message = deserialized_message->Serialize(&signers);
+  ASSERT_TRUE(serialized_message);
+  EXPECT_EQ(serialized_message, *message_bytes);
+  EXPECT_EQ(signers, std::vector<std::string>({kFromAccount}));
+
```

Comment:
I'll add more invalid test cases here to cover when byte arrays can't be serialized into message.

Reply:
Added in https://github.com/brave/brave-core/commit/31b92fe960375c71823d7e7d547ec18c0cec76aa